float32 left_vel
float32 right_vel
float32 left_current
float32 right_current
time time_stamp


